src/Servers/Kestrel/Core/src/Internal/Http2/Http2FrameWriter.cs

```diff
@@ -265,6 +265,23 @@ public ValueTask<FlushResult> WriteDataAsync(int streamId, StreamOutputFlowContr
             }
         }
 
+        public void WriteData(int streamId, StreamOutputFlowControl flowControl, in ReadOnlySequence<byte> data, bool endStream)
```

Comment:
nit: Not a fan of the name. Maybe CommitData?

Reply:
Hmm, it is calling into methods that are named WriteXXX. The methods are private though.

Reply:
WriteData is consistent. The only difference is the lack of flushing. WriteDataWithoutFlushing 😁?

Reply:
Renamed to make it explicit.